src/main/java/com/vertexvis/example/CreateAssemblyFromRevisionsExample.java

```diff
@@ -0,0 +1,131 @@
+package com.vertexvis.example;
+
+import com.vertexvis.ApiCallback;
+import com.vertexvis.ApiClient;
+import com.vertexvis.ApiException;
+import com.vertexvis.api.FilesApi;
+import com.vertexvis.example.cmdline.CommandLineOptions;
+import com.vertexvis.model.CreateFileRequest;
+import com.vertexvis.model.CreateFileRequestData;
+import com.vertexvis.model.CreateFileRequestDataAttributes;
+import com.vertexvis.model.FileMetadata;
+import com.vertexvis.model.Part;
+import com.vertexvis.model.PartDataRelationshipsPartRevisions;
+import picocli.CommandLine;
+
+import java.io.File;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.vertexvis.example.CallbackUtil.execute;
+
+public class CreateAssemblyFromRevisionsExample extends CommandLineOptions {
```

Comment:
Example to create new part revisions from uploaded files and then create an assembly from those existing part revisions